### DISCUSSION TITLE
Which <compilerSpecID> shall I use if I know the compiler is clang but in Linux?

### DISCUSSION THREAD

**Discussion by fzuo** [NationalSecurityAgency/ghidra#5576] [Q&A]
A program is compiled by clang but in Linux. I am confused which -cspec value shall I use. 
- compiler name="clang" spec="x86win.cspec" id="clangwindows"/
- compiler name="gcc" spec="x86gcc.cspec" id="gcc"/

I see the id for clang is 'clangwindows'. I am confused it is suitable for a program that is compiled/running in Linux. Or, is gcc also acceptable for this case? 

Thanks for your help in advance.

**Comment by astrelsky** [comment]
Choose the gcc compiler spec.
